ui/scss/component/_tags.scss

```diff
@@ -73,3 +73,10 @@
   margin: calc(2rem / 10) calc(2rem / 10);
   max-width: 20rem;
 }
+
+.tag--blocked-words {
+  .tag {
+    color: var(--color-text);
+    background-color: var(--color-gray-5);
```

Comment:
These will need custom variables in light/dark.scss. Try a lighter gray like `--color-gray-2` for light mode.